lib/sass/calcite-web/components/_form.scss

```diff
@@ -239,6 +240,32 @@
   }
   input[type='radio'] {
     -webkit-appearance: radio;
+    -webkit-border-radius: 50%;
+    border-radius: 50%;
+  }
+  // alignment fixes for edge
+  @supports (-ms-ime-align: auto) {
```

Comment:
we'll probably need to provide right-to-left styles here, as well

Reply:
Nice addition - just added.